docs/arcaflow/getting-started.md

```diff
@@ -1,101 +1,97 @@
 # Arcaflow Getting Started Guide
 
-## Step 1: Get a container engine
+## Running Workflows
+An Arcaflow workflow is a definition of steps structured together to perform complex actions. Workflows are defined as machine-readable YAML and therefore can be version-controlled and shared easily to run in different environments. A workflow is a way of encapsulating and sharing expertise and ensuring reproducible results.
```

Comment:
The intention here is to imply version-controlling the workflow itself. A workflow is a machine-readable definition, ergo it lends itself to version control. I don't really see the problem here.

Reply:
How do you version control any code? Sure there are tags for container images, but any code I maintain is version controlled by whatever mechanism makes sense. Hopefully that's something like git, but it could be file names.

As a workflow author, I fully intend to commit my workflows to a git repo and release tagged versions as appropriate. To me this is actually very critical functionality of Arcaflow as a workflow ~~engine~~ ~~system~~ tool.

Reply:
Yeah, but for Arcaflow it really is just that file: any version control *you* choose to apply is entirely beyond Arca's grasp, and I think that the phrasing here implies it's something Arca does or at least is aware of. I don't see any point: at best, it seems like meaningless obfuscation. Of course you can version control anything you want. If the reader understands that, nothing needs to be said. If they don't, this isn't going to help a bit and may add confusion.